app/uk/gov/hmrc/tariffclassificationfrontend/views/templates/ruling_template.scala.html

```diff
@@ -18,7 +18,7 @@
 @import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, Decision}
 @import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
 
-@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(c: Case, d: Decision)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
Changed to pass decision into the template so that I could just copy/paste the template body from trader version (to ensure they are kept in line)